DockerfileUbuntu

```diff
@@ -26,8 +26,6 @@ RUN mkdir -p /teddycloud/certs \
     && mkdir -p /teddycloud/data/www   \
     && mkdir -p /tmp
 
-RUN ln -s /teddycloud /etc/teddycloud
```

Comment:
I may add /teddycloud to the list of fixed path, so no symbolic link is needed

Reply:
In this proposal I'm changing the default path via Makefile parameter, so it works without adding it to the fixed path list.

Reply:
Got it. I'll leave it as it is for now